Jenkinsfile.yml

```diff
@@ -55,5 +58,77 @@ node('ccode'){
       print "Archive the artifacts"
       archiveArtifacts artifacts: 'client-sdk/tpm_*.tar.gz', fingerprint: true, allowEmptyArchive: false
     }
+    
+    try{
+      stage('Run SmokeTest'){
+        checkout([$class: 'GitSCM', branches: [[name: '*/master']], extensions: [[$class: 'RelativeTargetDirectory', relativeTargetDir: 'sdo-test']], userRemoteConfigs: [[credentialsId: 'sdo-automationgithubtoken', url: 'https://github.com/secure-device-onboard-ci/sdo-test']]])
+        sh '''
+          mkdir -p sdo-test/binaries/pri
+          mkdir -p sdo-test/binaries/supply-chain-tools
+          mkdir -p sdo-test/binaries/rendezvous-service
+          mkdir -p sdo-test/binaries/client-sdk/x86_ecdsa256_sct_bin
+          mkdir -p sdo-test/binaries/client-sdk/x86_ecdsa384_sct_bin
+          mkdir -p sdo-test/binaries/client-sdk/x86_epid_sct_bin
```

Comment:
Please remove EPID related commands. EPID support is being removed as part of PR #3